sdk/agent_diff_pkg/agent_diff/code_executor.py

```
@@ -0,0 +1,346 @@
+                f"{base_url}/api/env/{environment_id}/services/linear",
+            ),
+        ]
+
+    def _run_code(
+        self,
+        code: str,
+        command: list,
+        timeout: int = 30,
+    ) -> Dict[str, Any]:
+        try:
+            result = subprocess.run(
```

Ruff (0.14.2)

`subprocess` call: check for execution of untrusted input

(S603)